### DISCUSSION TITLE
无法通过电子邮件授权设备

### DISCUSSION THREAD

**Discussion by ltg1710** [getlantern/lantern-forum-cn#724] [Q&A]
在`通过电子邮件授权设备`页面，输入电子邮箱后，点击`提交`按钮无反应。检查电子邮箱也无相关邮件（已检查垃圾邮箱）。

我的账号：ltg1710@gmail.com

**Comment by 41766717** [comment]
同样碰到该问题

**Comment by MaxMKMax** [comment]
@ltg1710 @41766717 

您好，谢谢您的反馈。非常抱歉影响了您的体验感。谢谢您的等待与支持。

您可以帮助我们在应用程序中提交报告您的问题吗？需要用您受影响的版本和设备, 您可以访问Lantern应用程序并提交log/ 报告。

go to Support (转到支持)
select Report an Issue (选择报告问题)
choose issue type (选择问题类型)
include additional details (包括额外的详细信息)
hit Send Report (点击发送报告)

当您在app提交问题报告后您会收到邮件. 邮件标题上有 8 位数号码. 请回来这里留言您的log/ticket号码. 因为我需要拿您的反馈报告号码来和科技团队沟通您遇到的问题. 不然我无法提交您遇到的问题给科技团队. 不好意思, 需要您的帮忙. 谢谢!

有了这些资料，我们科技团队便可以尽快解决问题。您可以在留言区给我知道。谢谢。

**Reply by ltg1710** [reply]
@Mike777Khoo 我是通过windows11 家庭版来登录latern的，lantern为最新的preview版本。按照你说的流程报告问题，点击Send Report后，弹出
![image](https://github.com/user-attachments/assets/c3d588bd-ac09-4340-8ac4-efae5a2edd19)

**Comment by MaxMKMax** [comment]
@ltg1710 @41766717 你们用那个设备呢? 电脑还是手机遇到的问题?

**Reply by MaxMKMax** [reply]
@ltg1710 您好. 好的, 谢谢您的帮忙. 我会反馈给技术团队的.
